packages/shared/components/organisms/NetworkSideDrawer.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { isNetworkSideDrawerOpen } from '@core/network/stores'
+    import { SideDrawer, Text, TextType, Icon } from 'shared/components/'
+    import { Icon as IconEnum } from '@lib/auxiliary/icon'
+
+    function closeDrawer(): void {
+        $isNetworkSideDrawerOpen = false
+    }
+</script>
+
+{#if $isNetworkSideDrawerOpen}
```

Comment:
We'll likely need to come up with a more dynamic and robust solution, most likely something similar to what mobile is doing with its own drawers (reason being that we will need to have routers for the different flows).

For this PR, I think adding a `sideDrawerState` store (corresponding to `ISideDrawerState`) with just one attribute, `direction`, would be good (should also be encoded into a `SideDrawerDirection` enum). If the state is not `null` or `undefined` then we render the side drawer. I will think about this before we start adding all of the other views and how we can route them together :+1: